### ISSUE TITLE
Training on NYUV2 dataset

### ISSUE THREAD

**Issue by Fannzi** [ashok-arjun/Monocular-Depth-Estimation#11]
hi, I used the training code on the NYUV2 dataset, but in the wandb logs, the depth predictions images of the network were all zero.
The training code was originally pulled from this repository. hope you can help me out.
![image](https://user-images.githubusercontent.com/62327668/222385283-443d2c03-8592-4902-a3cb-c18efcdf7b27.png)

**Comment by Fannzi** [comment]
Changing the learning rate to a smaller value helps.
